src/main/java/io/github/treesitter/jtreesitter/Language.java

```diff
@@ -60,12 +60,9 @@ private static UnsatisfiedLinkError unresolved(String name) {
      * <h4 id="load-example">Example</h4>
      *
      * <p>{@snippet lang="java" :
-     * Language language;
-     * try (Arena arena = Arena.ofConfined()) {
-     *    String library = System.mapLibraryName("tree-sitter-java");
-     *    SymbolLookup symbols = SymbolLookup.libraryLookup(library, arena);
-     *    language = Language.load(symbols, "tree_sitter_java");
-     * }
+     * String library = System.mapLibraryName("tree-sitter-java");
+     * SymbolLookup symbols = SymbolLookup.libraryLookup(library, Arena.ofAuto());
```

Comment:
I hope `Arena.ofAuto()` is actually safe here and jtreesitter always holds a strong reference to the `Language`. Otherwise this could crash the JVM as well in case the `Arena` is not reachable anymore and is garbage collected (?).